save-cloud-charts/save-cloud/values-minikube.yaml

```diff
@@ -3,12 +3,16 @@
 profile: dev
 backend:
   profile: dev,secure,kubernetes
+  clusterIP: 10.106.182.9
```

Comment:
The original idea was to assign fixed IPs for backend and orchestrator, put them into `agent.properties` and then the agent would use them to connect to these services. However, it shouldn't actually be needed anymore, because agents are run as Kubernetes pods and should be able to connect to other services using DNS. I'll change this part